WordPress/src/main/AndroidManifest.xml

```diff
@@ -280,6 +281,30 @@
                     android:scheme="http" >
                 </data>
 
+                <data
```

Comment:
Just wanted to grumble about the fact we can't do something like `android:scheme="http|https"` here :)